__tests__/app.test.js

```diff
@@ -57,3 +57,24 @@ describe.only('GET /api/articles task 4',() => {
         })
     })
 });
+
+//5
+describe('GET /api/articles/:article_id task 5',() => {
+    test('responds with status code 200, appropriate object properties and values',() => {
```

Comment:
Great first test, for this one you will also need to test for errors:

- an invalid ID (eg. "not-an-id")
- a valid ID but the article doesn't exist (eg. 9999)